scripts/quickstart.sh

```diff
@@ -250,12 +263,26 @@ done
 sleep 0.5
 
 if [ -n "${GCS_BUCKET}" -o -n "${S3_ENDPOINT}" ]; then
-  ${THANOS_EXECUTABLE} tools bucket web \
+  ${THANOS_EXECUTABLE} bucket web \
```

Comment:
`bucket` command is under `tools`. This shouldn't change I believe.
Have you test it with the latest Thanos version?
```suggestion
  ${THANOS_EXECUTABLE} tools bucket web \
```

Reply:
Done